src/components/Slogan/Slogan.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+
+import './Slogan.css';
+
+export default function Slogan({ setSlogans }) {
+  const [currentSlogan, setCurrentSlogan] = useState('');
+  const handleSaveSlogan = (e) => {
+    e.preventDefault();
+    if (!currentSlogan) return;
+    setSlogans((prevSlogans) => [...prevSlogans, currentSlogan]);
```

Comment:
Nice work passing the callback function and using the spread operator to create the new array